### PR TITLE
SDK and tests for hybrid RP

### PR DESCRIPTION
**Pull request by farehar**
https://github.com/Azure/azure-rest-api-specs/pull/7568

### CODE REVIEW

File: sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HybridCompute</PackageId>
+    <Description>Provides developers with libraries for managing hybrid machines</Description>
+    <Version>0.0.1</Version>
+    <AssemblyName>Microsoft.Azure.Management.HybridCompute</AssemblyName>
+    <PackageTags>management;hybridcompute;</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+        This is private preview release of the Azure Hybrid Compute SDK.
```
Comment (isra-fel) [inline sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj]: Hey @farehar , are you sure if this is for private preview? It should be against azure/sdk-for-net-pr repo if so.

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj (hunk 1 above)
Comment (farehar) [reply sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj]: @isra-fel: You are right, this is for our public preview.

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @farehar ,

- could you put a link to the swagger review PR(s) in description
- As a public preview SDK, its spec should also be in the public spec repo. Please update `generate.ps1` so that it doesn't point to the private spec repo
- And you'll see a .txt file after running `generate.ps1`, please also check it in
- please also check in `AssemblyInfo.cs`, and make sure the versions in it are updated

Thanks

**Comment by isra-fel** [comment]
@farehar ping on this

**Comment by farehar** [comment]
@isra-fel : One of the feedback was to move our swagger spec to public repository. We weren't ready to do that last month. I've created a pull request in public repository today. If i address rest of the feedback, would we be able to merge before swagger spec PR is complete?

**Comment by isra-fel** [comment]
> @isra-fel : One of the feedback was to move our swagger spec to public repository. We weren't ready to do that last month. I've created a pull request in public repository today. If i address rest of the feedback, would we be able to merge before swagger spec PR is complete?

Nope, sorry. SDK should be generated based on merged spec.

**Comment by farehar** [comment]
@isra-fel : I've addressed all the feedback. Here's the link to our swagger spec in public repository https://github.com/Azure/azure-rest-api-specs/tree/master/specification/hybridcompute/resource-manager

**Comment by isra-fel** [comment]
@farehar `generate.ps1` needs to be checked in too

File: sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HybridCompute</PackageId>
+    <Description>Provides developers with libraries for managing hybrid machines</Description>
+    <Version>0.0.1</Version>
```
Comment (isra-fel) [inline sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj]: First version should be `0.1.0` please. And I believe your module will be in preview, right?
```suggestion
    <Version>0.1.0-preview</Version>
```

File: sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure Hybrid Compute Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Hybrid Compute Resources.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```
Comment (isra-fel) [inline sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyVersion("0.1.0.0")]
```

File: sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+[assembly: AssemblyFileVersion("1.0.0.0")]
```
Comment (isra-fel) [inline sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyFileVersion("0.1.0.0")]
```

**Review by isra-fel — COMMENTED** [review]
@farehar Please CIL thanks

File: sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj (hunk 2 above)
Comment (farehar) [reply sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj]: yes. updated.

File: sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (farehar) [reply sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs]: updated

File: sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs (hunk 4 above)
Comment (farehar) [reply sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs]: updated

**Review by isra-fel — APPROVED** [review]

**Pull request merged**